lib/ansible/modules/windows/win_iis_webbinding.py

```diff
@@ -28,74 +15,58 @@
 ---
 module: win_iis_webbinding
 version_added: "2.0"
-short_description: Configures a IIS Web site.
+short_description: Configures a IIS Web site binding.
 description:
      - Creates, Removes and configures a binding to an existing IIS Web site
 options:
   name:
     description:
       - Names of web site
     required: true
-    default: null
-    aliases: []
+    aliases: website
```

Comment:
`aliases: [website]` should fix the Shippable issue

Reply:
thanks